java/jre_ver.bzl

```diff
@@ -0,0 +1,9 @@
+def jre_ver(version):
+  """Extract JRE version from an OpenJDK debian package.
+    debian packages have versions in the form:
+       openjdk-8-jre*: 8u171-b11-1~deb9u1
+       openjdk-11-jre*: 11.0.1+13-2~bpo9+1
+  """
+  if version.startswith("8u"): return version.split("-",2)[0]
+  if version.startswith("11."): return version.split("+",2)[0]
```

Comment:
nit: I believe `version.split("+")[0]` (no optional second parameter) will work and look simpler. I don't think we need this kind of optimization.

Reply:
Agree. Perhaps the consumer can handle this on their own.